guides/source/active_storage_overview.md

```diff
@@ -36,10 +36,10 @@ files.
 ## Setup
 
 Active Storage uses two tables in your application’s database named
-`active_storage_blobs` and `active_storage_attachments`. After upgrading your
-application to Rails 5.2, run `rails active_storage:install` to generate a
-migration that creates these tables. Use `rails db:migrate` to run the
-migration.
+`active_storage_blobs` and `active_storage_attachments`. After creating a new
+application (or upgrading your application to Rails 5.2), run
```

Comment:
Shouldn't `rails app:update` run this automatically if `config/storage.yml` exists (or `options[:active_storage]` is enabled)?

Reply:
It did not on my fresh `--edge` generated app, which is how I stumbled itupon.